apache/trafficcontrol
# Use Chromium for Traffic Portal integration tests

## What does this PR (Pull Request) do?

Following up on discussion in https://github.com/apache/trafficcontrol/issues/5750#issuecomment-821568994, this PR makes the TP Integration Tests use Chromium instead of Google Chrome in both CDN in a Box (79ddc1344a) and GitHub Actions (69c8139700).

This PR also removes the requirement that `docker-compose.yml` is used in any command that uses `docker-compose.traffic-ops-test.yml` or `docker-compose.traffic-portal-test.yml` (a04fc6cf78) while still preserving the speedups from #5283.


## Which Traffic Control components are affected by this PR?


- CDN in a Box
    * TP Integration Tests
    * TO API Tests
- Documentation
- Traffic Portal (Integration Tests README)
- GitHub Actions

## What is the best way to verify this PR?

- Run the TP Integration Tests in CDN in a Box:
    ```shell
    docker-compose -f docker-compose.yml -f docker-compose.traffic-portal-test.yml up -d db edge trafficportal trafficops trafficvault portal-integration-test
    docker-compose -f docker-compose.traffic-portal-test.yml logs -f portal-integration-test
    ```

- Run the TO API Tests in CDN in a Box:
    ```shell
    docker-compose -f docker-compose.yml -f docker-compose.traffic-ops-test.yml up -d edge enroller dns db smtp trafficops trafficvault integration
    docker-compose -f docker-compose.traffic-ops-test.yml logs -f integration
    ```

- Check whether the *TP Integration Tests* GHA workflow passes

## The following criteria are ALL met by this PR


- [x] An update to CHANGELOG.md OR such an update is not necessary
